cmd/earthly/main.go

```diff
@@ -3664,6 +3744,203 @@ func (app *earthlyApp) actionSatelliteUnselect(cliCtx *cli.Context) error {
 	return nil
 }
 
+func (app *earthlyApp) actionProjectList(cliCtx *cli.Context) error {
+
+	orgName := cliCtx.String("org")
+	if orgName == "" {
+		return errors.New("org name is required")
+	}
+
+	cloudClient, err := cloud.NewClient(app.apiServer, app.sshAuthSock, app.authToken, app.console.Warnf)
+	if err != nil {
+		return errors.Wrap(err, "failed to create cloud client")
+	}
+
+	projects, err := cloudClient.ListProjects(cliCtx.Context, orgName)
+	if err != nil {
+		return errors.Wrap(err, "failed to list projects")
+	}
+
+	for _, project := range projects {
+		app.console.Printf("%s\n", project.Name)
+	}
+
+	return nil
+}
+
+func (app *earthlyApp) actionProjectRemove(cliCtx *cli.Context) error {
+	orgName := cliCtx.String("org")
+	if orgName == "" {
+		return errors.New("org name is required")
+	}
+
+	projectName := cliCtx.Args().Get(0)
+	if projectName == "" {
+		return errors.New("project name is required")
+	}
+
+	cloudClient, err := cloud.NewClient(app.apiServer, app.sshAuthSock, app.authToken, app.console.Warnf)
+	if err != nil {
+		return errors.Wrap(err, "failed to create cloud client")
+	}
+
+	err = cloudClient.DeleteProject(cliCtx.Context, orgName, projectName)
+	if err != nil {
+		return errors.Wrap(err, "failed to remove project")
+	}
+
+	return nil
+}
+
+func (app *earthlyApp) actionProjectCreate(cliCtx *cli.Context) error {
+	orgName := cliCtx.String("org")
+	if orgName == "" {
+		return errors.New("org name is required")
+	}
+
+	projectName := cliCtx.Args().Get(0)
+	if projectName == "" {
+		return errors.New("project name is required")
+	}
+
+	cloudClient, err := cloud.NewClient(app.apiServer, app.sshAuthSock, app.authToken, app.console.Warnf)
+	if err != nil {
+		return errors.Wrap(err, "failed to create cloud client")
+	}
+
+	_, err = cloudClient.CreateProject(cliCtx.Context, projectName, orgName)
+	if err != nil {
+		return errors.Wrap(err, "failed to create project")
+	}
+
+	return nil
+}
+
+func (app *earthlyApp) actionProjectMemberList(cliCtx *cli.Context) error {
+	orgName := cliCtx.String("org")
+	if orgName == "" {
+		return errors.New("org name is required")
+	}
+
+	projectName := cliCtx.Args().Get(0)
+	if projectName == "" {
+		return errors.New("project name is required")
+	}
+
+	cloudClient, err := cloud.NewClient(app.apiServer, app.sshAuthSock, app.authToken, app.console.Warnf)
+	if err != nil {
+		return errors.Wrap(err, "failed to create cloud client")
+	}
+
+	members, err := cloudClient.ListProjectMembers(cliCtx.Context, orgName, projectName)
+	if err != nil {
+		return errors.Wrap(err, "failed to list project members")
+	}
+
+	for _, m := range members {
+		app.console.Printf("%s\t%s\t%s\n", m.UserID, m.UserEmail, m.Permission)
```

Comment:
can you use the `tabwriter` package? see `actionOrgListPermissions` for an example of how it is used.

Reply:
SGTM